src/pages/discord.js

```diff
@@ -0,0 +1,11 @@
+import React from 'react';
+
+const Discord = () => {
+  React.useEffect(() => {
+    window.location.replace('https://discord.gg/SXreMyQ')
+  }, [])
+  
+  return null;
+};
```

Comment:
Interesting approach to doing this.
For static sites I've most commonly seen usage of meta as this works for noscript browsers.
```html
<meta http-equiv="refresh" content="0; url=http://example.com/">
```
Though not sure if going off the domain gives any extra CORS limitations.

Reply:
@Beanow Good point, changed to using meta-tags instead so it works for no script browsers

Reply:
Also, we can let the server handle it seems like.
https://docs.netlify.com/routing/redirects/#syntax-for-the-netlify-configuration-file

Reply:
I would rather handle it on the client side since server configuration isn't version controlled or visible to everyone. Makes it more portable too if we decide to move to something other than Netlify

Reply:
Isn't the netlify.toml something you can store in the repo?

Reply:
Yea, but it also couples us to Netlify vs having a generic solution. Might also cause weirdness if there's naming collisions or something with the routes defined on the client level

Reply:
Avoiding vendor lock-in is an argument I can get behind :P